vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -137,8 +137,10 @@
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\JSCRuntime.cpp">
       <ExcludedFromBuild>true</ExcludedFromBuild>
     </ClCompile>
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\jsi.cpp" />
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\JSIDynamic.cpp" />
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\jsi\jsi.cpp" />
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\jsi\JSIDynamic.cpp" />
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\jsi\jsilib-posix.cpp" />
```

Comment:
><ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\jsi\jsilib-posix.cpp" /> [](start = 4, length = 78)

Why do we need this file to be compiled as part of the Windows build?

Reply:
It had the macro blocks hence included .. but removed it now.. Thanks!!